test/dlc_tests/sanity/test_security_check.py

```diff
@@ -21,3 +27,38 @@ def test_security(image):
         run(f"{docker_exec_cmd} python /test/bin/security_checks.py ")
     finally:
         run(f"docker rm -f {container_name}", hide=True)
+
+
+@pytest.mark.model("N/A")
+@pytest.mark.integration("check OS dependencies")
+def test_ecr_scan(image, ecr_client):
+    """
+    Run ECR Scan Tool on an image being tested, and raise Error if vulnerabilities found
+    1. Start Scan.
+    2. For 5 minutes (Run DescribeImages):
+    3.1. If imageScanStatus == COMPLETE: exit loop
+    3.2. If imageScanStatus == IN_PROGRESS or AttributeNotFound(imageScanStatus): continue loop
+    3.3. If imageScanStatus == FAILED: raise RuntimeError
+    4. If DescribeImages.imageScanStatus != COMPLETE: raise TimeOutError
+    5. assert imageScanFindingsSummary.findingSeverityCounts.HIGH/CRITICAL == 0
+
+    :param image: str Image URI for image to be tested
+    :param ecr_client: boto3 Client for ECR
+    """
+    timezone = pytz.timezone("US/Pacific")
```

Comment:
Why bring timezone into this, can't this just be implemented with time.time()? Is the goal just to run a command for 5 minutes in a while loop? Also why are you sleeping for 1 second in each loop?

Is the goal to execute this 600 times with 1 second in between, or 600s? Right now, it seems like this will run an indeterminate amt of times due to the seconds implementation plus the sleep addition

Reply:
Good point. I added this timezone part to deal with checking the time elapsed between the last time the ECR Scan was run and the current time, to ensure that there is a 24 hour difference. But I modified the implementation to just catch the LimitExceededException that is caused by running it multiple times within 24 hours, instead.

This was left over from that implementation. I'll fix this.